src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -76,6 +76,9 @@ This package is an internal implementation of the .NET Core SDK and is not meant
       _InstallTargetingPackIntoLocalDotNet;
       _CreateTargetingPackArchive;
     </BuildDependsOn>
+
+    <!-- No-op when in source build -->
+    <BuildDependsOn Condition="'$(IsTargetingPackBuilding)' != 'false' and '$(DotNetBuildFromSource)' == 'true'"/>
```

Comment:
Delete because this additional definition of the target should override the previous one. Not needed anyhow.

Reply:
I don't quite follow? Removing this line breaks things. By default, this is not empty and there are targets that run.

Reply:
Is that coming in another iteration?

Reply:
I removed the condition above but will keep this one.